cmd/releasego/update-azure-linux.go

```diff
@@ -276,34 +276,20 @@ func updateSpecFile(assets *buildassets.BuildAssets, changelogDate time.Time, sp
 		oldVersion = goversion.New(matches[2])
 	}
 
-	var oldRelease int
+	var oldRelease string
 	if matches := specFileReleaseRegex.FindStringSubmatch(specFileContent); matches == nil {
 		return "", fmt.Errorf("no Release declaration found in spec content")
 	} else {
-		var err error
-		oldRelease, err = strconv.Atoi(matches[2])
-		if err != nil {
-			return "", fmt.Errorf("failed to parse Release number: %w", err)
-		}
+		oldRelease = matches[2]
```

Comment:
not sure of std in this Go code base, but if 2 is a magic (index) number, replace with a const?

Reply:
It's just the group number in the reg exp, but yeah, that could be stored in a constant adjacent to the exp itself.